.dockerignore

```diff
@@ -0,0 +1,11 @@
+/node_modules
+/coverage
+/build
+/bin
+.env.local
```

Comment:
Using `.env` to store github tokens would mean it's a secret. It should be added in `.gitignore` as well.

Reply:
I just copied all these from .gitignore

Reply:
added `.env`